packs/cloudflare/actions/get_ips.py

```diff
@@ -38,9 +38,8 @@ def run(self):
 
         url = "{}/client/v4/ips".format(self.API_HOST)
         payload = {}
-        headers = {"Content-Type": "application/json"}
 
-        data = self._get(url, headers, payload)
+        data = self._get(url, payload)
```

Comment:
Minor thing - next time please prefer keyword over regular arguments, easier to catch a programmer error in cases like this when an API (method signature) changes :)